src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx

```diff
@@ -153,6 +156,9 @@
   <data name="Fancy_Zones_Editor_App_Title" xml:space="preserve">
     <value>FancyZones Editor</value>
   </data>
+  <data name="Fancy_Zones_Main_Editor_Window" xml:space="preserve">
+    <value>FancyZones main editor window</value>

```

Comment:
The word `window` should be removed because the name of the control must not be present in the accessible name. Same for the Canvas layout editor window.
Ref - https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/basic-accessibility-information#accessible_name

on running accessibility insights - 
![image](https://user-images.githubusercontent.com/28739210/96628328-f3521700-12c6-11eb-9dca-ed9355d6976c.png)

Reply:
resolved